.github/workflows/test.yml

```diff
@@ -142,3 +142,18 @@ jobs:
       with:
         name: Test Reports
         path: build/reports/tests
+
+    - name: "Notify QA Slack"
+      if: steps.set-env-step.outputs.test-env == 'alpha' || steps.set-env-step.outputs.test-env == 'staging'
+      uses: broadinstitute/action-slack@v3.8.0
+      # see https://github.com/broadinstitute/action-slack
+      env:
+        GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+        SLACK_WEBHOOK_URL: ${{ secrets.SLACK_WEBHOOK_URL }}
+        MATRIX_CONTEXT: ${{ toJson(matrix) }}
+      with:
+        status: ${{ job.status }}
+        channel: "#dsde-qa"
+        username: "Workspace Manager tests"
+        text: "Integration tests"
```

Comment:
This says "integration tests" for unit tests, connected tests and integration tests. Perhaps `text` should be specific to the type of tests that are running? We could also just remove `text `or make it something more generic?

Reply:
@ansingh7115 good call, I removed the `text` parameter altogether!